CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to the Deploy to Azure extension will be documented in this
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/). Versioning follows an internal Azure DevOps format that is not compatible with SemVer.
 
+## 1.0.7
+### Added
+- Integrated with repo analysis and template service
+- If not a remote repo, have a provision of creating github repo
+
+## 1.0.6
```

Comment:
Did we release this version? If no, then fold it into 1.0.7

Reply:
we released this version, changes were done directly in release branch